static/css/components.css

```
@@ -339,6 +339,82 @@ button:disabled {
+}
+
+.loading-overlay-card .loading-spinner {
+    display: inline-block;
+    width: 40px;
+    height: 40px;
+    border-width: 3px;
+    margin-bottom: 1rem;
+}
+
+.loading-overlay-card h3 {
+    font-family: 'Outfit', sans-serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)